components/arm/client.go

```diff
@@ -144,19 +153,20 @@ func (c *client) IsMoving(ctx context.Context) (bool, error) {
 	return resp.IsMoving, nil
 }
 
-func getModel(ctx context.Context, r robotpb.RobotServiceClient, name string) referenceframe.Model {
+func getModel(ctx context.Context, r robotpb.RobotServiceClient, name string) (referenceframe.Model, bool, error) {
```

Comment:
[minor] not sure why this needs to change, can we not just return the error instead of the `bool error`?

Reply:
yes modified to log instead